streams/src/main/java/org/apache/kafka/streams/state/internals/MeteredVersionedKeyValueStore.java

```diff
@@ -198,6 +280,19 @@ protected void initStoreSerde(final StateStoreContext context) {
                 prepareValueSerde(plainValueSerde, new SerdeGetter(context))
             );
         }
+
+        private byte[] serializeAsBytes(final VersionedRecord<byte[]> versionedRecord) {
+            if (versionedRecord == null) {
+                return null;
+            }
+            final Serde<ValueAndTimestamp<byte[]>> VALUE_AND_TIMESTAMP_SERDE
```

Comment:
Why do we create a `Serde` -- if we only need the `Serializer` we can just use `new ValueAndTimestampSerializer`